capstone/capdb/models.py

```diff
@@ -24,7 +24,10 @@ def db_type(self, connection):
 
 def checksum(xml):
     m = hashlib.md5()
-    m.update(xml.encode())
+    hashstring = xml
```

Comment:
This should be unneeded with my related PR to correctly serialize xml -- can also drop changes to data/1.csv.gz and data/2.csv.gz